terraform/covidtest.usa.gov.tf

```diff
@@ -79,458 +79,278 @@ resource "aws_route53_record" "acme_challenge_eastb_covidtest_usa_gov_cname" {
-#     zone_id                = "Z166TLBEWOO7G0"
-#     evaluate_target_health = false
-#   }
-# }
-#
-# resource "aws_route53_record" "route_covidtest_usa_gov_a_east" {
-#   zone_id        = aws_route53_zone.usa_gov_zone.zone_id
-#   name           = "route.covidtest.usa.gov."
-#   type           = "A"
-#   set_identifier = "east"
-#   latency_routing_policy {
-#     region = "us-east-1"
-#   }
-#   alias {
-#     name                   = "east.covidtest.usa.gov."
-#     zone_id                = aws_route53_zone.usa_gov_zone.zone_id
-#     evaluate_target_health = true
-#   }
-# }
-# resource "aws_route53_record" "route_covidtest_usa_gov_aaaa_east" {
-#   zone_id        = aws_route53_zone.usa_gov_zone.zone_id
-#   name           = "route.covidtest.usa.gov."
-#   type           = "AAAA"
-#   set_identifier = "east"
-#   latency_routing_policy {
-#     region = "us-east-1"
-#   }
-#   alias {
-#     name                   = "east.covidtest.usa.gov."
-#     zone_id                = aws_route53_zone.usa_gov_zone.zone_id
-#     evaluate_target_health = true
-#   }
-# }
+resource "aws_route53_record" "covidtest_usa_gov_a" {
+  zone_id = aws_route53_zone.usa_gov_zone.zone_id
+  name    = "covidtest.usa.gov."
+  type    = "A"
+  alias {
+    name                   = "dncvgqeo9peie.cloudfront.net"
+    zone_id                = local.cloud_gov_cloudfront_zone_id
+    evaluate_target_health = false
+  }
+}
+resource "aws_route53_record" "covidtest_usa_gov_aaaa" {
+  zone_id = aws_route53_zone.usa_gov_zone.zone_id
+  name    = "covidtest.usa.gov."
+  type    = "AAAA"
+  alias {
+    name                   = "dncvgqeo9peie.cloudfront.net"
+    zone_id                = local.cloud_gov_cloudfront_zone_id
+    evaluate_target_health = false
+  }
+}
+
+# westb direct & health check
+resource "aws_route53_record" "westb_covidtest_usa_gov_a" {
+  zone_id = aws_route53_zone.usa_gov_zone.zone_id
+  name    = "westb.covidtest.usa.gov."
+  type    = "A"
+  alias {
+    name                   = "westb-domains-0-1443414084.us-gov-west-1.elb.amazonaws.com"
+    zone_id                = "Z33AYJ8TM3BH4J"
+    evaluate_target_health = false
+  }
+}
+resource "aws_route53_record" "westb_covidtest_usa_gov_aaaa" {
+  zone_id = aws_route53_zone.usa_gov_zone.zone_id
+  name    = "westb.covidtest.usa.gov."
+  type    = "AAAA"
+  alias {
+    name                   = "westb-domains-0-1443414084.us-gov-west-1.elb.amazonaws.com"
+    zone_id                = "Z33AYJ8TM3BH4J"
+    evaluate_target_health = false
+  }
+}
+resource "aws_route53_health_check" "westb_covidtest_usa_gov_health" {
+  fqdn              = "westb.covidtest.usa.gov"
+  port              = 443
+  type              = "HTTPS_STR_MATCH"
+  resource_path     = "/"
+  search_string     = "COVIDtests.gov"
+  failure_threshold = "2"
+  request_interval  = "30"
+}
+
+# route -> westb
+resource "aws_route53_record" "route_covidtest_usa_gov_a_wb" {
```

Comment:
we should set TTL here to 5 seconds per AWS suggestion

Reply:
do you have a doc link? I thought that you couldn't set TTL on an alias

Reply:
https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/resource-record-sets-values-multivalue.html#rrsets-values-multivalue-ttl

this is what makes me think you can. I'm testing in the cloud.gov account right now

Reply:
further googling shows this isn't possible